src/appleseed.studio/python/pythoninterpreter.h

```diff
@@ -46,10 +48,14 @@ class PythonInterpreter
     void execute_command(const char* command);
 
     void redirect_output(OutputRedirector redirector);
+    void set_mainwindow(MainWindow* mainWindow);
+    MainWindow* get_mainwindow();
```

Comment:
I'm not talking about returning a `const MainWindow*`, I'm just saying you should make `get_mainwindow()` itself const, which isn't the same thing:

    MainWindow* get_mainwindow() const;